main.js

```diff
@@ -1,12 +1,49 @@
 'use strict';
 
-const logger = require('ft-next-logger');
+const logger = require('ft-next-logger').logger;
+const fetchres = require('fetchres');
+const raven = require('raven');
+let ravenMiddleware;
+
+function sendErrorDev (err, req, res) {
+	if (err.name === fetchres.ReadTimeoutError.name) {
+		logger.error('Dependency timeout -', err);
```

Comment:
yeah, need to update the logs, but you can pass a error message string in the first argument, and an object in the second that will be turned into splunk fields, e.g.

```
logger.error('Whoops, there\'s been an error', { event: ' depedencytimeout' });
```